c_sharp_json_diff/DoubleDiffProcessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace c_sharp_json_diff
+{
+    class DoubleDiffProcessor : DiffProcessor
+    {
+        public override Dictionary<string, object> PerformDiffProcess(JToken left, JToken right)
+        {
+            string leftValueAsString = left.ToString();
+            string rightValueAsString = right.ToString();
+            Dictionary<string, object> dict = new Dictionary<string, object>();
+
+            if (!IsIntable(leftValueAsString, rightValueAsString))
+            {
+                NumericDiff diff = new NumericDiff();
+
+                if (double.TryParse(leftValueAsString, out double leftValueAsDouble) && double.TryParse(rightValueAsString, out double rightValueAsDouble))
+                {
+                    diff.Subtraction = leftValueAsDouble - rightValueAsDouble;
+                    diff.Division = rightValueAsDouble == 0 ? -1 : leftValueAsDouble / rightValueAsDouble;
+                }
+
+
+                if (diff.Subtraction != null && diff.Division != null)
+                {
+                    dict.Add("Subtraction", diff.Subtraction);
+                    dict.Add("Division", diff.Division);
+                }
+            }
+
+            return dict;
+        }
+
+        private bool IsIntable(string left, string right)
```

Comment:
Actually, I think we need to revisit this. The tests have made it clearer. Testing a try-parse on the two values seems to be too strict. For example, if the left is 20 and the right is 25, then the ratio right/left = 25/20 which is 5/4 or 1.25. This is clearly a double value, even though the two inputs are ints. 

So I would say we need to make ``IsIntable`` stricter, and probably rename it to something like ``IsIntDiffable``. We should still leave in the test that both left and right are ints. But to make it stricter, we should also confirm that ``right % left == 0``. That is, the remainder after division should be zero, meaning one divides perfectly into the other.

Reply:
As discussed, removed int processor, so this is no longer required